.github/workflows/prod-sync-odfe-cli.yml

```diff
@@ -0,0 +1,31 @@
+name: Prod Sync ODFE CLI Artifacts
+
+on:
+  repository_dispatch:
+    types: [prod-sync-odfe-cli] 
+
+jobs:
+  sync-odfe-cli-artifacts-prod:
+    name: sync-odfe-cli-artifacts-prod
+    runs-on: [ubuntu-18.04]
+    steps:
+    - uses: actions/checkout@v1
+
+    - name: Configure AWS Credentials
+      uses: aws-actions/configure-aws-credentials@v1
+      with:
+        aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+        aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+        aws-region: us-east-1
+
+    - name: S3 sync artifacts
+      run: |
+        #!/bin/bash
+        S3_BUCKET_PROD="artifacts.opendistroforelasticsearch.amazon.com"
+        S3_BUCKET_PLUGIN="artifacts.learning"
+        TAG_VERSION="$(jq '.["odfe-cli"].cutversion' ./bin/plugins.json |sed 's/"//g')"
```

Comment:
Just use plugins-info script.

Reply:
This seems simpler approach and as its not dependent on another script.

Reply:
Let's make it consistent since everything else is using that script.

Reply:
Everything else is using plugin_tag.sh but that uses odfe version for filter, which wont work for CLI. I can make the change to plugins-info script so that CLI and data prepper are in sync. I personally would go direct to grab the cut version as that flexibility is provided by json.

Reply:
made change to pull tag info from plugins-info